Makefile

```diff
@@ -15,13 +15,12 @@
 all: verify test
 
 dep:
-	cp .Gopkg.toml Gopkg.toml
-	-dep init
-	dep ensure
+	go mod tidy
 
 verify: dep
-	go get -u github.com/alecthomas/gometalinter
-	gometalinter --install
+	# todo gometalinter is DEPRECATED, Use https://github.com/golangci/golangci-lint
+	GO111MODULE=off go get -u github.com/alecthomas/gometalinter
+	GO111MODULE=off gometalinter --install
```

Comment:
Can this be enhanced to get a fixed version of `gometalinter` while you are touching this code? Like depending on "master" for Kubernetes, depending on master of gometalinter (or golangci-lint) has the risk that changes in those project(s) break this project.

I know that this is more complicated because (for reasons that I don't understand) `go get` in non-module mode doesn't support specifying a version, but perhaps it works when adding it as module and then building?

Reply:
with GO111MODULE=off its not easy, but we can fix it later by replacing with golangci-lint. In fact gometalinter is DEPRECATED, so its 'fixed'.